jwst/stpipe/tests/test_utilities.py

```diff
@@ -0,0 +1,34 @@
+"""Test utility funcs"""
+from jwst.stpipe.utilities import all_steps
+
+
+# Snapshot of known steps
+KNOWN_STEPS = set([
+    'AlignRefsStep','Ami3Pipeline', 'AmiAnalyzeStep', 'AmiAverageStep',
```

Comment:
Wow. Who/what/where attempts to keep this list up to date as we continue development and create/rename/remove step classes?

Reply:
This could be parred down to a short-list that will probably not change and would take less maintenance.

However, the intent of `all_steps` is to return all steps, and this should be expected to fail in create/delete cases.  In particular, as a canary-test, unintended changes to step definitions will be caught. As to maintenance, this would be no different than any other code change that requires unit test justification. If the value vs. work is considered insufficient, then, as stated, the test could be limited.